garage/tf/policies/maml_policy.py

```diff
@@ -0,0 +1,153 @@
+"""Maml Policy."""
+import tensorflow as tf
+
+from garage.core import Serializable
+from garage.tf.policies.base2 import StochasticPolicy2
+
+# This is an alternative variable store for maml.
+MAML_VARIABLE_STORE = set()
+MAML_TASK_INDEX = 0
+
+
+class MamlPolicy(StochasticPolicy2):
+    """
+    MamlPolicy used by Model-Agnostic Meta-Learning.
+
+    This policy take any garage policies that use models and augment
+    them with one step adaptation tensors.
+
+    Args:
+        wrapped_policy: A garage policy using model.
+        n_tasks: Number of task for meta learning.
+        adaptation_step_size: The step size for one step adaptation.
+        name: Name of this policy.
+    """
+
+    def __init__(self,
+                 wrapped_policy,
+                 n_tasks,
+                 adaptation_step_size=0.5,
+                 name="MamlPolicy"):
+
+        self.wrapped_policy = wrapped_policy
+        self.n_tasks = n_tasks
+        self.name = name
+        self._initialized = False
+        self._adaptation_step_size = adaptation_step_size
+        self._adapted_param_store = dict()
+        self._create_update_opts()
+
+        super().__init__(wrapped_policy._env_spec)
+        Serializable.quick_init(self, locals())
```

Comment:
Do we still need Serializable here?

Reply:
Good catch. Forgot to remove it. Thanks!